Dockerfile.oraclelinux7

```diff
@@ -0,0 +1,164 @@
+ENV OPENSSL_ROOT=/usr/local/ssl
+RUN --mount=type=cache,target=$DOWNLOADS_CACHE_DIR \
+    wget -N --progress=bar:force -P $DOWNLOADS_CACHE_DIR \
+        https://www.openssl.org/source/openssl-${OPEN_SSL_VERSION}.tar.gz \
+    && tar xf $DOWNLOADS_CACHE_DIR/openssl-${OPEN_SSL_VERSION}.tar.gz -C . \
+    && cd openssl-${OPEN_SSL_VERSION} \
+    && ./config --prefix=$OPENSSL_ROOT --openssldir=$OPENSSL_ROOT shared zlib \
+    && make -j$THREADS_ARG \
+    && make install \
+    && echo "$OPENSSL_ROOT/lib" >> /etc/ld.so.conf.d/openssl-${OPEN_SSL_VERSION}.conf \
+    && ldconfig -v \
+    && cd .. \
+    && rm -rf openssl-${OPEN_SSL_VERSION}
+    # Could also run `make test`
+    # `make install` places shared libraries into $OPENSSL_ROOT
+
+# Installing CMake
+# CMake will look for $OPENSSL_ROOT_DIR
+ENV OPENSSL_ROOT_DIR=$OPENSSL_ROOT
+RUN --mount=type=cache,target=$DOWNLOADS_CACHE_DIR \
+    source scl_source enable devtoolset-9 \
+    && wget -N --progress=bar:force -P $DOWNLOADS_CACHE_DIR \
+        https://github.com/Kitware/CMake/releases/download/v$CMAKE_VERSION/cmake-$CMAKE_VERSION.tar.gz \
+    && tar xzf $DOWNLOADS_CACHE_DIR/cmake-$CMAKE_VERSION.tar.gz -C . \
+    && cd cmake-${CMAKE_VERSION} \
+    && ./bootstrap --prefix=/usr/local \
+    && make -j$THREADS_ARG \
+    && make install \
+    && cd .. \
+    && rm -r cmake-${CMAKE_VERSION}
+
+RUN groupadd ${USER} --gid ${GID} \
+    && useradd -ms /bin/bash ${USER} --uid ${UID} --gid ${GID}
+
+WORKDIR /home/$USER
+
+RUN echo "source scl_source enable devtoolset-9" >> .bashrc
+
+# For the deployment of Java artifacts to the Maven repository
+COPY <<-"EOF" .m2/settings.xml
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <servers>
+        <server>
+            <id>${JenkinsHops.RepoID}</id>
+            <username>${JenkinsHops.User}</username>
+            <filePermissions>664</filePermissions>
+            <directoryPermissions>775</directoryPermissions>
+            <password>${JenkinsHops.Password}</password>
+        </server>
+    </servers>
+</settings>
+EOF
+
+RUN chown $USER:$USER -R .
+
+USER $USER:$USER
+
+# See https://stackoverflow.com/a/51264575/9068781 for conditional envs
+FROM rondb-build-dependencies as build-all
+ARG UID
+ARG GID
+
+ARG DEPLOY_TO_REPO
+ENV DEPLOY_ARG=${DEPLOY_TO_REPO:+-d}
+
+ARG RELEASE_TARBALL
+ENV RELEASE_ARG=${RELEASE_TARBALL:+-r}
+
+RUN mkdir rondb-src rondb-bin rondb-tarball
+RUN --mount=type=bind,source=.,target=rondb-src \
+    --mount=type=cache,target=rondb-bin,uid=$UID,gid=$GID,id=oracle7-rondb2104-bin \
+    source scl_source enable devtoolset-9 \
+    && rondb-src/build_scripts/release_scripts/build_all.sh \
+    -s rondb-src \
+    -b rondb-bin \
+    -o rondb-tarball \
+    -j $THREADS_ARG \
+    $DEPLOY_ARG $RELEASE_ARG
+
+# run with --output <output-folder>
```

Comment:
```
export BUILDKIT_PROGRESS=plain
docker buildx build . \
-f Dockerfile.oraclelinux7 \
--tag rondb-build-all:21.04.9 \
--target build-all \
--build-arg BUILD_THREADS=6 --output /tmp/

```

failed on my machine 

```
#27 exporting to client
#27 copying files
#27 copying files 307.51MB 5.0s
#27 copying files 643.92MB 10.0s
#27 copying files 654.41MB 10.3s done
#27 ERROR: error from receiver: failed to create device /tmp/dev/console: operation not permitted
------
 > exporting to client:
------
error: failed to solve: error from receiver: failed to create device /tmp/dev/console: operation not permitted
```

Reply:
The `--output` flag is only meant to be used for `--target get-package-all`. Can you try again?